### DISCUSSION TITLE
how to prevent ssr

### DISCUSSION THREAD

**Discussion by weepy** [sveltejs/kit#6064] [General]
With the new changes I was under the impression that I should be able to do something like this : 

```svelte
// +page.svelte
<script context="module">
    export const ssr = false
</script>
<script>
    console.log(window)
</script>
```

 => window is not defined

or 
```js
// +page.js
export const ssr = false
```

=> `export const ssr` has been removed

How can I turn off ssr for a particular page?

**Comment by david-plugge** [comment]
Seems like its still only possible within [handle](https://kit.svelte.dev/docs/hooks#handle).
Nothing else found in the docs regarding page level ssr config

**Comment by geoffrich** [comment]
This is still a TODO post the routing changes in #5778. For now, handle is the way to go.
